wagtail/admin/templates/wagtailadmin/bulk_actions/confirmation/form.html

```diff
@@ -0,0 +1,6 @@
+<form action="{{ submit_url }}" method="POST">
```

Comment:
You should also include the `non_field_errors` template

Reply:
@jacobtoppm this seems to have been added. below `{% include 'wagtailadmin/shared/non_field_errors.html' %}`

Reply:
Yep, I think this is resolved now!